index.js

```diff
@@ -1,9 +1,13 @@
 'use strict'
 
 const { createTerminus } = require('./lib/terminus')
-const { HealthCheckError } = require('./lib/terminus-error')
+const { HealthCheckError } = require('./lib/errors/health-check-error')
```

Comment:
should we combine these errors into a single error file?

Reply:
Will do